chroma-manager/tests/integration/core/api_testcase_with_test_reset.py

```diff
@@ -958,7 +968,10 @@ def wipe(x):
         self.execute_commands([wipe(x) for x in zfs_device_paths],
                               server0['fqdn'], 'wiping disks')
 
-        self.execute_simultaneous_commands(['partprobe', 'udevadm settle'], fqdns, 'sync partitions')
+        # only partprobe the devices we are cleaning, as we can get
+        # EBUSY for the root disk for example
+        self.execute_simultaneous_commands(['partprobe %s' % " ".join(zfs_device_paths),
```

Comment:
Seems like this could be extracted out into a function as it's being used 3 times. It could, for example, be called like this:
```
execute_simultaneous_commands(zfs_device_paths, fqdns)
```
In this way, you are passing in the paths of interest as well as the fqdns for each particular case.